docs/source/manim_directive.py

```diff
@@ -58,19 +58,45 @@ def construct(self):
         an image representing the last frame of the scene will
         be rendered and displayed, instead of a video.
 
+    seealso_classes
+        A list of classes, separated by spaces, that is
+        rendered in a SEEALSO block after the source code.
+
+    seealso_functions
+        A list of functions and methods, separated by spaces,
+        that is rendered in a SEEALSO block after the source code.
+
 """
 from docutils.parsers.rst import directives, Directive
 from docutils.parsers.rst.directives.images import Image
 
 import jinja2
 import os
 from os.path import relpath
+from typing import List
 
 import shutil
 
 classnamedict = {}
 
 
+def process_name_list(option_input: str, reference_type: str) -> List[str]:
+    r"""Reformats a string of space separated class names
+    as a list of strings containing valid Sphinx references.
+
```

Comment:
I think it would be nice if we can show the example of its use

Reply:
e.g. at line 32:
As a second application: Link from Gallery to Reference
 .. manim:: Example
    :quality: low	    
    :seealso_classes: Tex MathTex NumberPlane

    class Example(Scene):	    
        def construct(self):

Reply:
I've added an example of how to use it to the existing example.